Formula/qcachegrind.rb

```diff
@@ -1,15 +1,14 @@
 class Qcachegrind < Formula
   desc "Visualize data generated by Cachegrind and Calltree"
   homepage "https://kcachegrind.github.io/"
```

Comment:
Did they rename this to kcachegrind?

Reply:
It's the same project. `kcachegrind` uses KDE frameworks while `qcachegrind` only uses Qt.